requirements/common.txt

```diff
@@ -140,3 +137,6 @@ zope.interface==4.1.2
 
 # Django extension for static asset pipeline
 django-pipeline==1.6.8
+
+# Needed for iOS push notifications
+git+https://github.com/sharmaeklavya2/PyAPNs.git@remove-simplejson#egg=apns-2.0.1
```

Comment:
My pull request to remove simplejson from PyAPNs (https://github.com/djacobs/PyAPNs/pull/173) has been accepted. We can now directly use PyAPNs without relying on my fork.